app/messaging/create-submission-msg.js

```diff
@@ -219,7 +220,17 @@ function getEmailDetails (submission, desirabilityScore, notifyTemplate, agentAp
       productivityScore: getQuestionScoreBand(desirabilityScore.desirability.questions, 'Q19'),
       collaboration: submission.collaboration,
       collaborationScore: getQuestionScoreBand(desirabilityScore.desirability.questions, 'Q20'),
-      sssi: submission.sSSI
+      sssi: submission.sSSI,
+      businessName: submission.businessDetails.businessName,
+      farmerName: submission.farmerDetails.firstName,
+      farmerSurname: submission.farmerDetails.lastName,
+      agentName: submission.agentDetails.firstName ?? 'N/A',
+      agentSurname: submission.agentDetails.lastName ?? ' ',
+      farmerEmail: submission.farmerContactDetails.email,
+      agentEmail: submission.agentContactDetails.email ?? 'N/A',
+      contactConsent: submission.consentOptional ? 'Yes' : 'No',
+      scoreDate: (new Date(today.setMonth(today.getMonth() + 6))).toLocaleDateString('en-GB')
```

Comment:
Is this supposed to be +6 months? Or just today's date

Reply:
this happens when you copy paste without thinking .... Fixing it now